### PR TITLE
Fix location of oci hooks

### PR DESCRIPTION
**Pull request by rhatdan**
Please provide the following information:

- What did you do?

- How did you do it?

- How do I see it or verify it?

- A picture of a cute animal (not mandatory but encouraged)

Signed-off-by: Dan Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Probably best to fix the original patch, This needs to be fixed in 1.11 versions of rhel and fedora.

With this fix the oci-systemd-hook and oci-register-machine seem to work perfectly.  Only paramater needed to pass for a systemd based container is --stop-signal
@mrunalp @runcom PTAL

**Comment by runcom** [comment]
I'll fix the patch in 1.11 branches and 1.10.3 branches 

LGTM, I guess there's no need to merge here

File: README.md [hunk 1]
```diff
@@ -136,11 +136,12 @@ of starting and stopping of containers.  For example have a log agent that
...
-/usr/libexec/docker/hooks.d to search for hooks, if the directory exists
-docker will execute the executables in this directory via runc/libcontainer i
-using PreStart and PostStop.  It will also send the config.json file as the
-second paramater.
+Dockerhooks reads directory in either /usr/libexec/oci/hooks.d to search for hooks,
+if the directory exists docker will execute the executables in this directory via
+runc/libcontainer using PreStart and PostStop.  It will also send the config.json
+file as the second paramater. These hooks allow us to use oci-systemd-hook and
+oci-register-machine hooks, which register the docker container with the host, and
+allow you to run systemd as pid1 inside of a container without requiring --privilege.
```
Comment (runcom) [inline README.md]: Typo "privileged"

**Comment by rhatdan** [comment]
Right just close this patch once you fix the other patches, and fix my --privileged flag.

**Comment by runcom** [comment]
ack

**Comment by runcom** [comment]
applied to fedora-1.10.3 and rhel-1.10-3 (1.11 branches are still WIP, too much changes going on)

**Comment by mrunalp** [comment]
:+1:

**Pull request closed without merging**